### ISSUE TITLE
Microphone not available with new Pebble 2

### ISSUE THREAD

**Issue by Noxmiles** [k-yle/Pebble-Wikipedia#2]
On the new Pebble 2 the Symbol for searching with the microphone is not available. Instead this button is blank (see pic). You can only use random and location.

![pebble-wiki](https://cloud.githubusercontent.com/assets/10343510/20195968/e69235ba-a798-11e6-9761-d6a3926da32d.jpg)

PS: Also, some merging errors are in the source code.
@k-yle

**Comment by rasierra** [comment]
i don't believe this issue is 'done' as i am still experiencing the same problem with my pebble 2.

**Comment by OmegaRogue** [comment]
Me too
